src/components/PersonalInformation/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { useLocation } from "react-router-dom";
+import DateInput from "../DateInput/index.jsx";
+import TextInput from "../TextInput/index.jsx";
+import styles from "./styles/styles.module.scss";
+
+const PersonalInformation = () => {
+  const location = useLocation();
+  let section = <ul></ul>;
+
+  if (location.pathname === "/") {
+    section = (
+      <ul className={styles.list}>
+        <li>
+          <TextInput label="Name" minlength={2} isRequired={true} />
```

Comment:
If the default is true, you don't have to pass it.

Reply:
done